src/main/java/racingcar/Constant.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+public class Constant {
+
+    public static final int RANDOM_RANGE = 10;
+    public static final int MINIMUM_MOVE = 4;
+    public static final int START_POSITION = 0;
+    public static final int MAXIMUM_NAME_LENGTH = 5;
+    public static final int MINIMUM_NATURAL_NUMBER = 1;
+
+    public static final String POSITION_FLAG = "-";
+    public static final String JOIN_REGEX = ", ";
+    public static final String COMMA_REGEX = ",";
+    public static final String CAR_NAME_PATTERN_REGEX = "^[a-zA-Z,]*$";
+    public static final String COIN_PATTERN_REGEX = "^[0-9]*$";
+
+    public static final String NOT_ALPHA_AND_COMMA_ERROR = "[ERROR]: 잘못된 입력입니다.(입력은 영어와 쉼표로만 이루어져있습니다.)";
+    public static final String NOT_ALLOW_LENGTH_ERROR = "[ERROR]: 잘못된 입력입니다.(입력은 5자 이하만 가능합니다.)";
+    public static final String DUPLICATE_NAME_ERROR = "[ERROR]: 잘못된 입력입니다.(같은 이름의 자동차는 두번 사용할 수 없습니다.)";
+    public static final String NOT_NUMERIC_ERROR = "[ERROR]: 잘못된 입력입니다.(시도 횟수는 숫자만 가능합니다.)";
+    public static final String MINIMUM_NATURAL_NUMBER_ERROR = "[ERROR]: 잘못된 입력입니다.(1 이상의 값만 입력이 가능합니다.)";
```

Comment:
예외를 다루는 건 어려워요 😂

왜냐하면 사용자의 입장에서 생각해야하기 때문이죠! 게다가 에러 메시지는 추후 사용자에게 직접 보여지거나 협업하는 개발자들이 확인할 수 있기에 잘 적어줘야합니다.

사용자가 느끼기에 A라는 행동을 했는데 그 행동이 왜 잘못되었고 어떻게 행동해야 사용자가 원하는 결과를 얻을 수 있을지를 잘 표현할 수 있어야 한다고 생각해요.

지금과 같은 상황에서는 충분하게 컨텍스트를 표현할 수 있다고 생각해요. 사용자로서 에러가 났을 때에 정상 행동을 취할 수 있는 단서를 제공하니까요!


다만 아래와 같이 에러 메시지는 일관된 톤 앤 매너를 가지면 좋을 것 같아요.
하나의 예시입니다) 입력이라는 단어가 중복되어서 줄여봤구요. 가능/불가능으로 어미를 통일해봤어요.

```suggestion
    public static final String NOT_ALPHA_AND_COMMA_ERROR = "[ERROR]: 잘못된 입력입니다.(자동차 이름은 영어와 쉼표로만 가능합니다.)";
    public static final String NOT_ALLOW_LENGTH_ERROR = "[ERROR]: 잘못된 입력입니다.(자동차 이름은 5자 이하만 가능합니다.)";
    public static final String DUPLICATE_NAME_ERROR = "[ERROR]: 잘못된 입력입니다.(자동차 이름은 중복이 불가능합니다.)";
    public static final String NOT_NUMERIC_ERROR = "[ERROR]: 잘못된 입력입니다.(시도 횟수는 숫자만 가능합니다.)";
    public static final String MINIMUM_NATURAL_NUMBER_ERROR = "[ERROR]: 잘못된 입력입니다.(시도 횟수는 1 이상의 값만 가능합니다.)";
```